src/main/java/org/isf/lab/gui/ExamPicker.java

```diff
@@ -279,7 +279,8 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
             }
         });
 
-        jButtonQuit.setText(MessageBundle.getMessage("angal.common.cancel"));
+	    jButtonQuit.setText(MessageBundle.getMessage("angal.common.cancel.btn"));
+	    jButtonQuit.setMnemonic(MessageBundle.getMnemonic("angal.common.cancel.btn.key"));
```

Comment:
this button maybe needs refactoring as well

Reply:
Thanks; done.